enterprise/install-scripts/openshift.ks

```diff
@@ -1363,7 +1373,7 @@ configure_rhn_channels()
     fi
     need_node_repo && repos+=('rhel-x86_64-server-6-ose-2.2-node' 'jb-ews-2-x86_64-server-6-rpm')
     need_client_tools_repo && repos+=('rhel-x86_64-server-6-ose-2.2-rhc')
-    need_jbosseap_cartridge_repo && repos+=('rhel-x86_64-server-6-ose-2.2-jbosseap' 'jbappplatform-6-x86_64-server-6-rpm')
+    need_jbosseap_cartridge_repo && repos+=("rhel-x86_64-server-${jbosseap_version}-ose-2.2-jbosseap" "jbappplatform-${jbosseap_version}-x86_64-server-6-rpm")
```

Comment:
I meant 'rhel-x86_64-server-6-ose-2.2-jbosseap' should be left unchanged, because ose only have 'rhel-x86_64-server-6-ose-2.2-jbosseap' channel, if here use "rhel-x86_64-server-${jbosseap_version}-ose-2.2-jbosseap", when user set jbosseap_version to 6.3, the scripts would try to look for 'rhel-x86_64-server-6.3-ose-2.2-jbosseap', but this channel does not exist.